### PR TITLE
chore: remove dead code from Batcher

### PR DESCRIPTION
**Pull request by dev7355608**
##### Description of change

Perhaps this isn't dead code, but just something under construction.

##### Pre-Merge Checklist

- [ ] Tests and/or benchmarks are included
- [ ] Documentation is changed or added
- [ ] Lint process passed (`npm run lint`)
- [ ] Tests passed (`npm run test`)

### CODE REVIEW

File: src/rendering/batcher/shared/Batcher.ts [hunk 1]
```diff
@@ -122,18 +122,11 @@ export class Batcher
 
     public dirty = true;
 
-    public batchIndex = 0;
```
Comment (dev7355608) [inline src/rendering/batcher/shared/Batcher.ts:125]: Always 0.

File: src/rendering/batcher/shared/Batcher.ts [hunk 2]
```diff
@@ -122,18 +122,11 @@ export class Batcher
...
-    public batches: Batch[] = [];
```
Comment (dev7355608) [inline src/rendering/batcher/shared/Batcher.ts:126]: Always empty.

File: src/rendering/batcher/shared/Batcher.ts [hunk 3]
```diff
@@ -122,18 +122,11 @@ export class Batcher
...
-
     // specifics.
     private readonly _vertexSize: number = 6;
 
     private _elements: BatchableObject[] = [];
 
-    private readonly _batchPool: Batch[] = [];
```
Comment (dev7355608) [inline src/rendering/batcher/shared/Batcher.ts:133]: Always empty.

File: src/rendering/batcher/shared/Batcher.ts [hunk 4]
```diff
@@ -122,18 +122,11 @@ export class Batcher
...
-    private _batchPoolIndex = 0;
-    private readonly _textureBatchPool: BatchTextureArray[] = [];
```
Comment (dev7355608) [inline src/rendering/batcher/shared/Batcher.ts:135]: Always empty.

**Comment by dev7355608** [comment]
Would we use `BigPool` here or a custom implementation?

**Comment by GoodBoyDigital** [comment]
lets use the local ones for this instance - mainly as its hot code and avoids any array modifications. In fact one sec - will push up a PR!

**Comment by Zyie** [comment]
closing in favor of #10711

**Comment by dev7355608** [comment]
We still need to remove `batchIndex` and `batches`, which are dead code.

**Pull request closed without merging**